install/installer/trafodion_config_default

```diff
@@ -125,6 +125,9 @@ export PASSWORD="admin"
 # hadoop cluster name
 export CLUSTER_NAME=""
 
+#Trafodion Cluster name, normally the same as Cloudera cluster name
+export CLUSTERNAME=""
```

Comment:
CLUSTERNAME is used through out Trafodion and it can not be changed. It was deleted awhile back, since it was believed to not being used and had some bad consequences.